test/sqllogictest/string.slt

```diff
@@ -863,3 +863,67 @@ SELECT ''::pg_catalog.CHAR(0)
 
 query error length for type char must be within \[1-10485760\], have 10485761
 SELECT ''::pg_catalog.CHAR(10485761)
+
+### position ###
+statement ok
+CREATE TABLE positiontest (strcol1 char(15), strcol2 char(15), vccol1 varchar(15), vccol2 varchar(15))
+
+statement ok
+INSERT INTO positiontest VALUES ('om', 'Thomas', 'om', 'Thomas'), ('foo', 'barbar', 'foo', 'barbar'),
+    (NULL, 'str', NULL, 'str'), ('str', NULL, 'str', NULL), ('释手', '爱不释手', '释手', '爱不释手'),
+    ('', 'str', '', 'str'), ('str', '', 'str', '')
+
+# invalid input
+statement error
```

Comment:
Could you add the error string after `statement error` in these tests? (Like [this](https://github.com/MaterializeInc/materialize/blob/main/test/sqllogictest/aggregates.slt#L18).)

Reply:
Absolutely!